doc/i18n.md

```diff
@@ -86,16 +86,15 @@ Opening a new branch
 Occasionally we create a new upstream branch for e. g. a new stable RHEL
 release. Run these steps to initialize translations for it:
 
- * On the [Zanata Cockpit project page](https://fedora.zanata.org/project/view/cockpit),
-   open the … menu in "Versions" and create a "New Version". Give it a name
-   that matches the new branch, and import existing translations from master
-   (or another branch if that's more appropriate). This will take a while to
-   get initialized.
+ * In [cockpit-weblate](https://github.com/cockpit-project/cockpit-weblate) create a new branch (ideally with the same name).
```

Comment:
Please fix the overly long line.

Also, let's just say "a new branch with the same name". There's no reason for wiggle room there, it would just be very confusing and break our import scripts.

Reply:
Updated.